### PR TITLE
Condense Bravo's Brew effects

### PR DESCRIPTION
**Pull request by pedrogrullada**

### CODE REVIEW

File: build/uuid-redirects.json [hunk 1]
```diff
@@ -84,6 +85,7 @@
     "Compendium.pf2e.equipment-effects.Item.vOgD9wfStLX1utte": "Compendium.pf2e.equipment-effects.Item.Effect: Skyrider Sword",
     "Compendium.pf2e.equipment-effects.Item.vKooOkXHvtqCgZYg": "Compendium.pf2e.equipment-effects.Item.Effect: Energized Cartridge",
     "Compendium.pf2e.equipment-effects.Item.vOU4Yv2MyAfYBbmF": "Compendium.pf2e.equipment-effects.Item.Effect: Aeon Stone Resonance (Amplifying)",
+    "Compendium.pf2e.equipment-effects.Item.wTZnKkT0K4Tdy8mD": "Compendium.pf2e.equipment-effects.Item.Effect: Bravo's Brew",
```
Comment (stwlam) [inline build/uuid-redirects.json:88]: Deleted 3 but only redirecting 2?

Comment (pedrogrullada) [reply build/uuid-redirects.json:88]: > Deleted 3 but only redirecting 2?

The "new" one is just the existing lesser one, repurposed

File: packs/equipment-effects/effect-bravos-brew.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                                    {
+                                        "not": "parent:origin:item"
+                                    }
+                                ]
+                            }
+                        ],
+                        "value": "Compendium.pf2e.equipment-srd.Item.Bravo's Brew (Greater)"
+                    }
+                ],
+                "key": "ChoiceSet",
```
Comment (simonward) [inline packs/equipment-effects/effect-bravos-brew.json:67]: Missing label

Comment (pedrogrullada) [reply packs/equipment-effects/effect-bravos-brew.json:67]: Do you mean the prompt? I did forget that

Comment (simonward) [reply packs/equipment-effects/effect-bravos-brew.json:67]: Yeah, sorry.

Comment (pedrogrullada) [reply packs/equipment-effects/effect-bravos-brew.json:67]: No worries, just making sure. Fixed!

**Pull request merged**